manifests/cert.pp

```diff
@@ -1,6 +1,7 @@
 define letsencrypt_client::cert (
-  String $webroot,
-  String $domain_name = $title,
+  $webroot,
```

Comment:
Unfortunately because I don't want to have to deal with maintaining the pre-puppet 4.0 truthiness of different types of variables we cannot remove the typing that is in the defined class parameters.

For more information about this particular problem see the document on comparing strings between Puppet 3.x and Puppet 4.x: http://docs.puppetlabs.com/puppet/4.3/reference/lang_updating_manifests.html#empty-strings-in-boolean-context-are-true